packages/phaserx/src/utils/tween.ts

```diff
@@ -31,10 +31,10 @@ export async function tween(config: TweenBuilderConfig, options?: { keepExisting
 }
 
 export function removeAllTweens(gameObject: Phaser.GameObjects.GameObject) {
-  // Access some hidden tween manager properties because tweenManager.getTweensOf does not return tweens in _add
-  // eslint-disable-next-line @typescript-eslint/no-explicit-any
-  const tweenManager = gameObject.scene.tweens as any;
-  if (!tweenManager) return;
-  const allTweens = [...tweenManager._add, ...tweenManager._pending, ...tweenManager._active] as Phaser.Tweens.Tween[];
-  for (const tween of allTweens) if (tween.hasTarget(gameObject)) tween.stop();
+  const tweenManager = gameObject.scene.tweens;
+  for (const tween of tweenManager.tweens) {
```

Comment:
<img width="711" alt="image" src="https://user-images.githubusercontent.com/508855/207946197-f0069231-cf7b-4c32-92af-edabe8e43236.png">

Not sure if this includes all the tweens (add, pending, active) but I assume it would given the description?

Reply:
the previous implementation was there due to a very obscure and hard to reproduce bug in Ember, where creatures were flying over the map (while they where not supposed to). Our assumption was that the game objects still had active tweens when they where reused for another entity (as part of the object pool), and executed the previous stale tween before the next one. This seemed to have been fixed by manually removing stopping tweens in all sub arrays (`_add`, `_pending`, `_active`).

It seems like these internal subarrays don't exist anymore in the tween manager, so I'd assume they're all included in that tween array.

Down to merge this change, just putting this context here in case the flying creatures appear again.

Also, maybe we don't need this custom logic anymore and could use `getTweensOf` from the tween manager? https://github.com/photonstorm/phaser/blob/v3.60.0-beta.10/src/tweens/TweenManager.js#L855